spec/features/admin/edit_organisation_details_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'features/support/sign_up_helpers'
+
+describe 'editing an organisations details' do
+  let!(:admin_user1) { create(:user, organisation: organisation) }
+  let!(:admin_user2) { create(:user) }
```

Comment:
This second admin user isn't relevant to most of the specs. I'd sink it into the spec on l35 (`"...tries to change another organisation..."`).